functions/Invoke-DbaAdvancedInstall.ps1

```diff
@@ -182,6 +182,7 @@ function Invoke-DbaAdvancedInstall {
     }
     if ($Credential) {
         $restartParams.Credential = $Credential
+        $restartParams.WsmanAuthentication = 'CredSSP'
```

Comment:
This would in all cases require CredSSP authentication. What if end user decides to manually specify Kerberos in -Authentication? Would it make sense to switch to using `$Authentication` parameter here?

Reply:
Good point, changed that.